src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -709,6 +713,8 @@ case class JesBackend(actorSystem: ActorSystem)
   }
 
   override def findResumableExecutions(id: WorkflowId)(implicit ec: ExecutionContext): Future[Map[ExecutionDatabaseKey, JobKey]] = {
-    globalDataAccess.findResumableJesExecutions(id)
+    globalDataAccess.findResumableExecutions(id, IsResumable, BuildJobKey)
   }
+
+  override def executionInfoKeys: Seq[String] = Seq("JES_RUN_ID", "JES_STATUS")
```

Comment:
People have a tendency to assume that `Seq()` is really just creating something which only conforms to the Seq interface. However inside of pattern matches and stuff like that scala knows what the object's type is for reals. Since people are generally used to Seq == List what has happened is that people supply List-specific operators in pattern matches which works until a non-List gets passed in. And of course since people love using Seq/Traversable as function signatures you have no way of knowing what *really* got passed in.

Really the larger point is that one should think about what they really want to be using and if it's "a bag of stuff" and you're just using Seq, just make it a List since that's what it really is and it's more clear what you're doing.